tests/test_subgraph/test_tokens.py

```diff
@@ -0,0 +1,35 @@
+from helpers.enums import Network
+from subgraph.queries.tokens import fetch_across_balances
+from tests.test_subgraph.test_data import ACROSS_BALANCES_TEST_DATA
+from copy import deepcopy
+
+def test_fetch_across_balances(mocker):
+    mocker.patch(
+        "subgraph.subgraph_utils.Client.execute",
+        side_effect=[
```

Comment:
Just a point. If you need to return static value, better use `return_value` instead of `side_effect`. I used `side_effect` in test harvests because I need for gql client to return different value each time it is called